### ISSUE TITLE
readme document example code

### ISSUE THREAD

**Issue by jimtal001** [LCAV/pyroomacoustics#89]
I just wanted to mention that in the pyroomacoustics example code of the readme document you have one typo that is a bit confusing:

```
# Create a linear array beamformer with 4 microphones
# with angle 0 degrees and inter mic distance 10 cm
R = pra.linear_2D_array([2, 1.5], 4, 0, 0.04)  <--------- should the 0.04 be changed to 0.1 for 10 cm ?
```

Thanks

**Comment by fakufaku** [comment]
Thanks for spotting that!

**Comment by fakufaku** [comment]
Fixed in 84b2918.
